app/assets/stylesheets/application.css

```diff
@@ -1,3 +1,290 @@
 /*
  *= require font-awesome
  */
+
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;500;800&family=Poppins:wght@300;400&display=swap');
+
```

Comment:
- [ ] Great job adding user name validation 👏🏻 Let's now validate all user input (password, ...) to make sure that anyone with bad intentions cannot compromise their app.